nin/backend/socket.js

```diff
@@ -39,8 +39,10 @@ function socket(projectPath, onConnectionCallback) {
       switch (event.type) {
         case 'set':
           var data = {};
-          data[event.data.field] = event.data.value;
-          layers.update(projectPath, event.data.id, data, function (err) {
+          for(var key in event.data.fields) {
```

Comment:
Man i wish github could let me highlight regions for commenting.
Anyways, isn't this part equivalent to just passing event.data.fields directly to graph?